File: server/order-service/src/main/java/com/mshop/orderservice/controller/order/dto/AddressDto.java
package com.mshop.orderservice.controller.order.dto;

import com.mshop.orderservice.repository.entity.order.Location;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class AddressDto {
    private String addressId;

    private int provinceID;

    private int districtID;

    private String wardCode;

    private String provinceName;

    private String districtName;

    private String wardName;

    private String address;

    private Location location;

    private String fullAddress;

    public AddressDto(String s, int i, int i1, int i2, String ninhBình, String thànhPhốNinhBình, String phườngĐôngThành, Location location, String s1) {
    }
}

File: server/auth-service/src/main/kotlin/com/mshop/authservice/dto/AddressDto.kt
package com.mshop.authservice.dto

data class AddressDto(
    private var addressId: String? = "",

    val provinceID: Int = 0,

    val districtID: Int = 0,

    val wardCode: Int = 0,

    val provinceName: String = "",

    val districtName: String = "",

    val wardName: String = "",

    val address: String = "",

    val location: com.mshop.authservice.dto.LocationDto = com.mshop.authservice.dto.LocationDto(),

    val fullAddress: String = "",
)